content/500-platform/10-about.mdx

```diff
@@ -5,13 +5,133 @@ metaDescription: 'Learn about the Platform Console to integrate the Prisma Data
+
+Here's an overview of the number of projects per workspace on each plan:
+
+|              | **Starter** | **Pro** | **Business** | **Enterprise** |
+| ------------ | ----------- | ------- | ------------ | -------------- |
+| **Projects** | 5           | 10      | 15           | Custom         |
+
+### Environment
+
+An environment is an isolated space to provision PDP products for a given project. Environments typically map to development stages, like `dev`, `staging` or `prod`.
+
+In each environment, you can:
+
+- enable, disable and manage PDP products (Pulse, Accelerate, ...).
+- for **Accelerate**:
+  - set your database connection string.
+  - generate API keys.
+  - change the connection pool size.
+  - enable static IPs.
+- for **Pulse**:
+  - set your database connection string (of a [Pulse-compatible database](/pulse/database-setup)).
+  - configure the _region_ where Pulse is running.
+  - configure database replication.
+  - generate API keys.
+
+The number of environments you can create in a project depends on the [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) configured in your workspace:
+
+Here's an overview of the number of environments per project on each plan:
+
+|                  | **Starter** | **Pro**     | **Business** | **Enterprise** |
+| ---------------- | ----------- | ----------- | ------------ | -------------- |
+| **Environments** | 2 / project | 6 / project | 12 / project | Custom         |
+
+## Billing
+
+The [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) you select in your workspace determins how many projects and environments you can create in that workspace:
+
+|                  | **Starter** | **Pro**     | **Business** | **Enterprise** |
+| ---------------- | :---------- | ----------- | ------------ | -------------- |
+| **Projects**     | 5           | 10          | 15           | Custom         |
+| **Environments** | 2 / project | 6 / project | 12 / project | Custom         |
+
+### Per-workspace billing
+
+Billing in PDP is done on a per-workspace basis:
+
+- A pricing plan is selected per workspace. That means, your PDP account can have multiple workspaces where each workspace uses a different pricing plan.
+- A payment method is selected per workspace. That means, your PDP account can have multiple workspaces where each workspace has a different payment method.
+
+At the end of the billing period, **your selected payment method will be charged with the usage costs of the PDP products across _all_ the projects (and their environments) in that workspace**.
+
+You can configure all billing details in the **Billing** section of your project.
+
+### Prorated billing
+
+All our paid plans are prorated, which means **you're only billed for the duration you use a specific plan**.
+
+For example:
+
+- if you subscribe to our **Pro** plan for only 15 days in a 30-day month, you'll only be charged for half of the plan's price.
```

Comment:
How about taking this bullet point as subscribing in the middle of the month, while the last one is about cancelling/downgrading mid-month?

```suggestion
- if you subscribe to our **Pro** plan on the 15th day of a month, you'll only be charged the base plan price for the days left in that month.
```